### ISSUE TITLE
Story Brainstorming

### ISSUE THREAD

**Issue by JammingJammerIsJammingToJam** [JammingJammerIsJammingToJam/space-trading-sim#12]
Either improve the story I have set out or try writing your own but I think I've made a solid start on it so I'd prefer improvements.

**Comment by AdrianAgache** [comment]
I would say that the character is knocked back and hits their head instead of just randomly getting amnesia

**Comment by AdrianAgache** [comment]
Maybe the character is a cyborg so they are affected by Anomaly 31

**Comment by JammingJammerIsJammingToJam** [comment]
> I would say that the character is knocked back and hits their head instead of just randomly getting amnesia. It would make more sense.

Yeah that could work but what forces would be acting upon them if it's a strong energy wave? I was thinking along the lines of the energy overloading the brain.

**Comment by JammingJammerIsJammingToJam** [comment]
> Maybe the character is a cyborg so they are affected by Anomaly 31

Cyborg part comes at end of story as previously discussed

**Comment by AdrianAgache** [comment]
Ty
